app/models/organisation.rb

```diff
@@ -33,56 +32,59 @@ class Organisation < ActiveRecord::Base
-      select('organisations.*, COUNT(*) as site_count').
-      joins(
-         'INNER JOIN (
-            SELECT organisation_id, site_id FROM organisations_sites
-            UNION
-            SELECT s.organisation_id, s.id FROM sites s
-        ) AS organisations_sites ON organisations_sites.organisation_id = organisations.id').
-      joins('INNER JOIN sites ON sites.id = organisations_sites.site_id').
-      group('organisations.id'). # Postgres will accept a group by primary key
-      having('COUNT(*) > 0')
-    }
+  scope :with_sites, -> {
+    select('organisations.*, COUNT(*) as site_count').
+    joins('
+      INNER JOIN (
+        SELECT organisation_id, site_id FROM organisations_sites
+        UNION
+        SELECT s.organisation_id, s.id FROM sites s
+      ) AS organisations_sites ON organisations_sites.organisation_id = organisations.id
+    ').
+    joins('INNER JOIN sites ON sites.id = organisations_sites.site_id').
+    group('organisations.id'). # Postgres will accept a group by primary key
+    having('COUNT(*) > 0')
+  }
 
   # Returns organisations ordered by descending error count across
   # all their sites.
-  scope :leaderboard, -> { select(<<-postgreSQL
-    organisations.title,
-    organisations.whitehall_slug,
-    COUNT(*)                                     AS site_count,
-    SUM(site_mapping_counts.mapping_count)       AS mappings_across_sites,
-    SUM(unresolved_mapping_counts.mapping_count) AS unresolved_mapping_count,
-    SUM(error_counts.error_count)                AS error_count
-  postgreSQL
-  ).joins(<<-postgreSQL
-    INNER JOIN sites
-           ON sites.organisation_id = organisations.id
-    LEFT JOIN (SELECT sites.id AS site_id,
-                      COUNT(*)::integer AS mapping_count
-               FROM   mappings
-                      INNER JOIN sites
-                              ON sites.id = mappings.site_id
-               GROUP  BY sites.id) site_mapping_counts ON site_mapping_counts.site_id = sites.id
-    LEFT JOIN (SELECT sites.id AS site_id,
-                      COUNT(*)::integer AS mapping_count
-               FROM   mappings
-                      INNER JOIN sites
-                              ON sites.id = mappings.site_id
-               WHERE  mappings.type = 'unresolved'
-               GROUP  BY sites.id) unresolved_mapping_counts ON unresolved_mapping_counts.site_id = sites.id
-    LEFT JOIN (SELECT hosts.site_id AS site_id,
-                      SUM(count)::integer AS error_count
-               FROM   daily_hit_totals
-                      INNER JOIN hosts
-                              ON hosts.id = daily_hit_totals.host_id
-               WHERE  daily_hit_totals.http_status = '404'
-               AND daily_hit_totals.total_on >= (current_date - 30)
-               GROUP BY site_id) AS error_counts ON error_counts.site_id = sites.id
-    postgreSQL
-  ).group('organisations.id').order('error_count DESC NULLS LAST') }
+  scope :leaderboard, -> {
+    select(%{
```

Comment:
That one was actually me, the linter messed up the indentation so I "fixed" it.  I'll move back to a heredoc.